configs/tdh-test.yml

```diff
@@ -49,190 +49,6 @@ usage:
     monthly_growth: 0  # guess (4 cases for every new person case)
     start_with: 19000000   # from ES case index (total docs)
 
-  # Number of cases updated per user per month
-  case_transactions:
```

Comment:
Keep the case_transactions stuff for a bit of buffer, but reduce the start_with to # of cases they have